### DISCUSSION TITLE
Validating instance types for RDS

### DISCUSSION THREAD

**Discussion by evilensky** [aws-cloudformation/cfn-lint#2609] [Q&A]
Would cfn-lint catch something like trying to deploy Aurora mysql 8.0 on an unsupported instance type?

Edit: The X to the Y is we'd like to validate this exact scenario before we attempt a CF deploy, without keeping a matrix of support ourselves. Wondering if inserting this into our CD would make sense, for this specific scenario and any other matrixes of supported instances (Elasticache?)

**Comment by PatMyron** [comment]
[`scripts/update_specs_from_pricing.py`](https://github.com/aws-cloudformation/cfn-lint/blob/main/scripts/update_specs_from_pricing.py) maintains [`data/AdditionalSpecs/RdsProperties.json`](https://github.com/aws-cloudformation/cfn-lint/blob/main/src/cfnlint/data/AdditionalSpecs/RdsProperties.json) which is used by [rule `E3025`](https://github.com/aws-cloudformation/cfn-lint/blob/main/src/cfnlint/rules/resources/rds/InstanceSize.py) to validate that for RDS

Matrix validation isn't yet covered for other AWS database services. Those services just validate supported engines and instance types separately without knowing unsupported combinations
